src/TgSharp.Tests/packages.config

```diff
@@ -1,3 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="System.Reactive" version="4.3.2" targetFramework="net472" />
```

Comment:
what's the dep that needs System.Reactive?

Reply:
Websocket-client